
PLAY [Product-XBET__APP_FTP] ***************************************************

TASK [nginx_sync : Notify] *****************************************************
ok: [13.75.121.150] => {
    "msg": "Syncing Product-XBET__APP_FTP"
}
ok: [119.9.105.164] => {
    "msg": "Syncing Product-XBET__APP_FTP"
}
ok: [23.102.228.135] => {
    "msg": "Syncing Product-XBET__APP_FTP"
}

TASK [nginx_sync : include_vars] ***********************************************
ok: [13.75.121.150]
ok: [119.9.105.164]
ok: [23.102.228.135]

TASK [nginx_sync : Synchronizing Nginx] ****************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [13.75.121.150] (item=conf) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf/ ''", "item": "conf", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [23.102.228.135] (item=conf) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf/ ''", "item": "conf", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [119.9.105.164] (item=conf) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf/ ''", "item": "conf", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [13.75.121.150] (item=conf.d) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf.d/ ''", "item": "conf.d", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [119.9.105.164] (item=conf.d) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf.d/ ''", "item": "conf.d", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [23.102.228.135] (item=conf.d) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/conf.d/ ''", "item": "conf.d", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [13.75.121.150] (item=data) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/data/ ''", "item": "data", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [23.102.228.135] (item=data) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/data/ ''", "item": "data", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [119.9.105.164] (item=data) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/data/ ''", "item": "data", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [23.102.228.135] (item=html) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/html/ ''", "item": "html", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [13.75.121.150] (item=html) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/html/ ''", "item": "html", "msg": "filedescriptor out of range in select()", "rc": 257}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ValueError: filedescriptor out of range in select()
failed: [119.9.105.164] (item=html) => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --delete-after --copy-links --archive '--rsh=/usr/bin/ssh -S none -o Port=2556 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' '--out-format=<<CHANGED>>%i %n%L' /srv/Product-XBET__APP_FTP/nginx/html/ ''", "item": "html", "msg": "filedescriptor out of range in select()", "rc": 257}
	to retry, use: --limit @/ansible/nginx/site.retry

PLAY RECAP *********************************************************************
119.9.105.164              : ok=2    changed=0    unreachable=0    failed=1   
13.75.121.150              : ok=2    changed=0    unreachable=0    failed=1   
23.102.228.135             : ok=2    changed=0    unreachable=0    failed=1   


